main-serve-actor-singleton.py

```diff
@@ -14,13 +14,17 @@
 @ray.remote
 class SingletonProfile:
     def __init__(self) -> None:
-        self.profile = DatasetProfile("")
+        self.profile = DatasetProfile().view()
 
-    def add_profile(self, profile: DatasetProfile):
+    def add_profile(self, profile: DatasetProfileView):
         self.profile = self.profile.merge(profile)
 
     def get_summary(self):
-        return str(self.profile.to_summary())
+        try:
+            return str(self.profile.to_pandas())
+        except:
+            # TODO wait for this bug to be fixed
+            return []
 
 
```

Comment:
Need this to be released first: https://github.com/whylabs/whylogs/issues/864

Reply:
its fixed and released in version 1.1.3 or higher